### DISCUSSION TITLE
Truenas Scale Update(s)?

### DISCUSSION THREAD

**Discussion by dbb12345** [guydavis/machinaris#639] [Q&A]
Hi,
Looking to swap over to Mach and can not seem to find any version history and looks like it has not been updated on TNS?  Know of any place to find a revision history?  Going to swap as soon as they update to the newest Mach version.  I see a few old posts (January) that are using the version that is currently listed on TNS.  PS thank you for this tool and am looking forward to checking it out.

**Comment by guydavis** [comment]
Hi, yes sorry about the poor support for deployment on TrueNAS Scale.  Unfortunately, the app packagers at TrueNAS seem to have chosen to pin a particular version of Machinaris in their application template, rather than using the [floating tag](https://github.com/guydavis/machinaris/wiki/Releases#release-streams) `:latest`.  This means the TrueNAS version of Machinaris often lags far behind that available on all other platforms.  

Please drop into the [Discord](https://discord.gg/mX4AtMTt87) and check out the #truenas channel to get the latest tips from users on that platform.  Alternatively, you can use `docker-compose` on [any other platform quite easily](https://www.machinaris.app/). Hope this helps.
